Base/Python/slicer/util.py

```diff
@@ -3797,10 +3797,10 @@ def pip_uninstall(requirements):
 
 
 def longPath(path):
-    """Make long paths work on Windows, where the maximum path length is 260 characters.
+    r"""Make long paths work on Windows, where the maximum path length is 260 characters.
 
     For example, the files in the DICOM database may have paths longer than this limit.
-    Accessing these can be made safe by prefixing it with the UNC prefix ('\\?\').
+    Accessing these can be made safe by prefixing it with the UNC prefix ('\\\\?\\').
```

Comment:
Not sure this is correct - now that it's a raw string should not be escaping the backslashes with more backslashes.

Reply:
My understanding is that using raw literals avoid on level of escaping.

These are still needed so that sphinx can properly render it ... see before/after in the pull-request description https://github.com/Slicer/Slicer/pull/7390#issue-1993785459

To avoid this, we would need to use sphinx/docutils literals as well. See https://docutils.sourceforge.io/docs/ref/rst/roles.html#literal

Reply:
ah, I see - okay that makes sense.